neo/rawio/openephysbinaryrawio.py

```diff
@@ -213,31 +213,35 @@ def _parse_header(self):
                         )
 
                     # # If available, use 'states' to compute event duration
-                    if 'states' in info and info["states"].size:
-                        states = info["states"]
-                        timestamps = info["timestamps"]
-                        labels = info["labels"]
-                        rising = np.where(states > 0)[0]
-                        falling = np.where(states < 0)[0]
-
-                        # infer durations
-                        durations = None
-                        if len(states) > 0:
-                            # make sure first event is rising and last is falling
-                            if states[0] < 0:
-                                falling = falling[1:]
-                            if states[-1] > 0:
-                                rising = rising[:-1]
-
-                            if len(rising) == len(falling):
-                                durations = timestamps[falling] - timestamps[rising]
-
-                        info["rising"] = rising
-                        info["timestamps"] = timestamps[rising]
-                        info["labels"] = labels[rising]
-                        info["durations"] = durations
-                    else:
-                        info["durations"] = None
+                    info["durations"] = None
+                    # 'states' was introduced in OpenEphys v0.6. For previous versions, events used 'channels_states'
```

Comment:
```suggestion
                    # 'states' was introduced in OpenEphys v0.6. For previous versions, events used 'channel_states'
```
super minor, but if someone looks in the future just making sure the key is without the extra 's' for the comment.